src/frontend/ExtensionManager/index.module.scss

```diff
@@ -1,11 +1,21 @@
 .mmContainer {
   position: fixed;
-  right: 357px;
+  top: 0px;
+  right: 0px;
+  left: 0px;
+  bottom: 0px;
   display: flex;
   flex-direction: row;
   gap: 20px;
   z-index: 100000;
   top: var(--top-navbar-height);
+  background-color: transparent;
+  border: none;
+  > div {
```

Comment:
I guess this because we cannot update the nested div with a class right? Since ideally we would have a class, as at this case it may go to each and every div nested

Reply:
updated the selector